File: API/Models/Entities/ServiceInventoryItem.cs
namespace API.Models.Entities
{
    public class ServiceInventoryItem
    {
        public int Id { get; set; }
        public int ServiceId { get; set; }
        public int InventoryItemId { get; set; }
        public int QuantityNeeded { get; set; }
        public virtual InventoryItem? InventoryItem { get; set; }
        public virtual Service? Service { get; set; }
    }
}

File: Client/src/app/models/serviceInventoryItem.ts
import { InventoryItem } from "./inventoryItems";
import { Service } from "./service";

export interface ServiceInventoryItem {
    id?:              number;
    serviceId?:       number;
    inventoryItemId?: number;
    quantityNeeded?:  number;
    inventoryItem?:   InventoryItem;
    service?: Service;
}
